tests/test_parser/test_properties/test_init.py

```diff
@@ -31,13 +31,35 @@ def test_get_type_string(self, mocker):
         assert p.get_type_string(no_optional=True) == base_type_string
 
         p = Property(name="test", required=False, default=None, nullable=True)
-        assert p.get_type_string() == f"Union[Unset, Optional[{base_type_string}]]"
+        assert p.get_type_string() == f"Union[Unset, None, {base_type_string}]"
```

Comment:
Honestly I like this style better overall. I think the `Optional` name gets confusing in some cases (the whole reason we're changing this behavior...).